civictechprojects/views.py

```diff
@@ -355,6 +357,50 @@ def volunteer_with_project(request, project_id):
     return HttpResponse(status=200)
 
 
+# TODO: Pass csrf token in ajax call so we can check for it
+@csrf_exempt
+def renew_volunteering_with_project(request, application_id):
+    if not request.user.is_authenticated():
```

Comment:
It's not a big piece of code duplication but if the steps in lines 362-370 and then 386-393 gets more complicated in the future we might want to pull it into its own function to avoid reuse?